buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersionHelper.kt

```diff
@@ -52,11 +20,20 @@ object IntellijVersionHelper {
       return listOf(versionKey)
     }
 
-    val versionValue = instance[versionKey] ?: return emptyList()
-    if (versionValue is List<*>) {
-      return versionValue as List<String>
+    return when (versionKey) {
+      "earliestSupportedMajor" -> listOf(IntellijVersions.earliestSupportedMajor)
+      "latestMinorsOfOldSupportedMajors" -> IntellijVersions.latestMinorsOfOldSupportedMajors
+      "latestStable" -> listOf(IntellijVersions.latestStable)
+      "latestSupportedMajor" -> listOf(IntellijVersions.latestSupportedMajor)
+      "buildTarget" -> listOf(IntellijVersions.buildTarget)
+      "eapOfLatestSupportedMajor" -> {
```

Comment:
Or https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/list-of-not-null.html